scripts/docstrings_checker.py

```diff
@@ -176,8 +176,8 @@ def docstringify(docstring):
 class GoogleDocstring(_check_docs_utils.GoogleDocstring):
 
     re_multiple_type = _check_docs_utils.GoogleDocstring.re_multiple_type
-    re_param_line = re.compile(r"""
-        \s*  \*{{0,2}}(\w+)             # identifier potentially with asterisks
+    re_param_line = re.compile(
+        r"""\s*  \*{{0,2}}(\w+)         # identifier potentially with asterisks
```

Comment:
Why are we changing this? Ideally, the check should not consider this as a docstring.  If it does, we might need to modify the check instead of modifying the string here.

Reply:
I've been looking into this and it struck me that there should be only single quotes throughout all python files right? So perhaps the string should be changed to use single quotes then?

Reply:
Nope, we have double quotes in docstrings, see our [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python).

Reply:
But this is not a docstring right? Isn't it a multi line regex pattern?

Reply:
That is true, I don't think we've codified the case where we have multi-line regex. Though Google does [prefer](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#310-strings) double quotes over single, so I'll defer this to @apb7.

Reply:
@lilithxxx: No, I think we have codified the case for triple-quote [here](https://github.com/oppia/oppia/blob/develop/.pylintrc#L14). Only string-quote should be single, triple-quote and docstring-quote should be double. Since this is triple-quote, it should be double.

Reply:
@apb7 done!